src/templates/_includes/forms/editableTable.html

```diff
@@ -39,48 +40,48 @@
                                     {% include "_includes/forms/date" with {
-                                        name: cellName,
-                                        value: value,
-                                        disabled: static
+                                    name: cellName,
+                                    value: value,
+                                    disabled: static
                                     } only %}
                                 {%- case 'lightswitch' -%}
                                     {% include "_includes/forms/lightswitch" with {
-                                        name: cellName,
-                                        on: value,
-                                        value: col.value ?? 1,
-                                        small: true,
-                                        disabled: static
+                                    name: cellName,
+                                    on: value,
+                                    value: col.value ?? 1,
+                                    small: true,
+                                    disabled: static
                                     } only %}
                                 {% case 'select' -%}
                                     {% include "_includes/forms/select" with {
-                                        class: 'small',
-                                        name: cellName,
-                                        options: cell.options ?? col.options,
-                                        value: value,
-                                        disabled: static
+                                    class: 'small',
+                                    name: cellName,
+                                    options: cell.options ?? col.options,
+                                    value: value,
+                                    disabled: static
                                     } only %}
                                 {%- case 'time' -%}
                                     {% include "_includes/forms/time" with {
-                                        name: cellName,
-                                        value: value,
-                                        disabled: static
+                                    name: cellName,
+                                    value: value,
+                                    disabled: static
                                     } only %}
                                 {%- default -%}
-                                    <textarea name="{{ cellName }}" rows="1"{% if static %} disabled{% endif %}{% if col.placeholder is defined %} placeholder="{{ col.placeholder }}"{% endif %}>{{ value }}</textarea>
+                                    <textarea name="{{ cellName }}" rows="1"{% if static %} disabled{% endif %}{% if col.placeholder is defined %} placeholder="{{ col.placeholder }}" {% endif %}>{{ value }}</textarea>
```

Comment:
Why the extra space?

Reply:
Php storm said "There should be a space between attribute and previous attribute".